variables.tf

```diff
@@ -37,6 +37,12 @@ variable "kinesis_stream_name" {
   default     = ""
 }
 
+variable "cloudtrail_in_master" {
+  description = "Specifies whether cloudtrail_s3_bucket_name/kinesis_stream_name are in master account or not"
```

Comment:
I think we should have two separate flags for clarity.
For now we can just fix bucket issue. Ignore Kinesis.
I am sure anyone testing Kinesis RN